adcm_client/objects.py

```diff
@@ -296,29 +272,30 @@ def config(self, full=False):
 
     @allure_step("Save config")
     def config_set(self, data):
+        config = self.config(full=True)
```

Comment:
No needs for config read here. It is not used in "then" part of "if" in line 276.

Reply:
Yes you are right.